Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2074,29 +2103,38 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
         event.stop();
     }
 
-    _updateLayoutBadgeStatus()
-    {
-        if (!this._layoutBadgeElement)
-            return;
-
-        let hasVisibleOverlay = this.representedObject.layoutOverlayShowing;
-        this._layoutBadgeElement.classList.toggle("activated", hasVisibleOverlay);
-
-        if (hasVisibleOverlay) {
-            let color = this.representedObject.layoutOverlayColor;
-            let hue = color.hsl[0];
-            this._layoutBadgeElement.style.borderColor = color.toString();
-            this._layoutBadgeElement.style.backgroundColor = `hsl(${hue}, 90%, 95%)`;
-            this._layoutBadgeElement.style.setProperty("color", `hsl(${hue}, 55%, 40%)`);
-        } else
-            this._layoutBadgeElement.removeAttribute("style");
+    _updateBadges()
+    {
+        for (let [badge, badgeElement] of this._elementForBadge) {
+            switch (badge) {
+            case WI.DOMNode.LayoutFlag.Grid:
+            case WI.DOMNode.LayoutFlag.Flex: {
```

Comment:
Should these be `WI.DOMTreeElement.Badge.*` values to match the types used to insert the badge elements into the map? (Yes, they have the same values, but probably still better to be consistent.)